src/main/java/seedu/address/logic/filters/Filters.java

```diff
@@ -21,23 +21,27 @@ public static AbstractFilter getCorrespondingFilter(String info) {
         ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(" " + info + " ", PREFIX_NAME,
             PREFIX_EMAIL,
             PREFIX_PHONE,
-            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED);
+            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED, PREFIX_COE_EXPIRY);
 
 
-        if (argumentMultimap.getTotalSize() != 2) { // since there is also a dummy position :(
+        if (argumentMultimap.getTotalSize() > 2) { // since there is also a dummy position :(
+            System.out.println(argumentMultimap);
```

Comment:
Are we just leaving this debug code in?

Reply:
I mean I suppose we could, but I want to make sure, given the other debug code you left in

Reply:
hey, my bad. have removed the debug codes